src/toil/common.py

```diff
@@ -93,7 +93,7 @@ def __init__(self):
         self.nodeOptions = None
         self.minNodes = None
         self.maxNodes = [10]
-        self.alphaPacking = 0.0
+        self.alphaTime = 1800
```

Comment:
I think lower would be fine, but how much lower I'm not sure. My gut tells me 60 would be too low, because the per-node overhead becomes a larger proportion of cost as they run for less time. Basically, because it can take a decent amount of time (IIRC over a minute) just to download and launch the toil container, lower alphaTime means higher cost. I'm not sure there is really an "optimal" value of alphaTime, but intuitively I think the cost will increase massively as the value goes below a few minutes.

As a totally contrived example, imagine that you are running 1000 jobs that take 1 minute to run and take up exactly one instance each, which costs (say) 1 cent per minute. Also assume the toil container takes one minute to initialize after the instance boots (IIRC this is in the right ballpark). Then running this with different values of alphaTime would yield different costs*:

- alphaTime = 60: 1000 instances launched for 2 minutes each, $20.00
- alphaTime = 120: 500 instances launched for 3 minutes each, $15.00
- alphaTime = 600: 100 instances launched for 11 minutes each, $11.00

[*] This is under the simplifying assumption that there is only one scaler decision made, at the start: in reality the last two costs will be different (lower, but with noticably longer runtime; see the first point under 'considerations' up above). Unfortunately calculating the costs with extra scaler decisions is kind of a pain.